submissions/drewzag/hooli-style-popup/index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Hooli-style-Popup</title>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header__menu">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="#" class="header__link">Mail</a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">Images</a>
+          </li>
+          <li class="header__item">
+            <input type="checkbox" class="header__checkbox" id="checkbox" />
+            <label for="checkbox" class="label__checkbox">
+              <img
+                src="./popup-icons/popup-button.png"
+                alt="open or close popup menu"
```

Comment:
DEcorative images no need populated `alt` attribute, icon is a decorative image.)